CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 A declarative library for application development using cloud services.
 
+## 0.10.0
```

Comment:
Would this be for 0.10.1 since 0.10.0 is out already??

Reply:
Oof - yep that's what I meant to do. Let me update.